src/main/java/org/snomed/snowstorm/rest/ExportController.java

```diff
@@ -38,12 +39,19 @@ public class ExportController {
 	@Autowired
 	private ModuleDependencyService moduleDependencyService;
 
+	@Autowired
+	private ExecutorService executorService;
+
 	@Operation(summary = "Create an export job.",
 			description = "Create a job to export an RF2 archive. " +
 					"The 'location' response header contain the URL, including the identifier, of the new resource.")
 	@PostMapping
-	public ResponseEntity<Void> createExportJob(@Valid @RequestBody ExportRequestView exportConfiguration) {
-		String id = exportService.createJob(exportConfiguration);
+	public ResponseEntity<Void> createExportJob(@Valid @RequestBody ExportRequestView exportRequestView) throws InterruptedException {
+		String id = exportService.createJob(exportRequestView);
+		if (exportRequestView.isPersist()) {
```

Comment:
If `persist` is set to true, the export will automatically start. By default it is false so there should be no regressions.